app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -299,6 +303,30 @@ public boolean isHealthCodeExportEnabled() {
     public void setHealthCodeExportEnabled(boolean enabled) {
         this.healthCodeExportEnabled = enabled;
     }
+    
+    /** {@inheritDoc} */
+    @DynamoDBMarshalling(marshallerClass = StringIntegerMapMarshaller.class)
+    @Override
+    public Map<String,Integer> getMinSupportedAppVersions() {
+        return minSupportedAppVersions;
+    }
+
+    @Override
+    public void setMinSupportedAppVersions(Map<String,Integer> map) {
+        this.minSupportedAppVersions = (map == null) ? new HashMap<>() : map;
```

Comment:
Yeah, I get that. I was following the other examples of setters for things that shouldn't be set to null.